File: 151002/Kragel/lab1/src/main/java/by/bsuir/dc/repository/CreatorRepository.java
package by.bsuir.dc.repository;

import by.bsuir.dc.entity.Creator;
import by.bsuir.dc.repository.common.InMemoryCrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public class CreatorRepository extends InMemoryCrudRepository<Creator> {
}

File: 151003/Shafarenko/lab3/cassandra/Publisher/Repository/Implementation/CreatorRepository.cs
using Publisher.Entity.Db;
using Publisher.Repository.Implementation.Common;
using Publisher.Repository.Interface;
using Publisher.Storage.Common;

namespace Publisher.Repository.Implementation
{
    public class CreatorRepository(DbStorage context) : AbstractCrudRepository<Creator>(context), ICreatorRepository
    {
        public News GetByNewsId(int newsId)
        {
            throw new NotImplementedException();
        }

        public Task<News> GetByNewsIdAsync(int newsId)
        {
            throw new NotImplementedException();
        }
    }
}
